src/pages/wallet/Transfer.vue

```diff
@@ -14,15 +14,28 @@ import { nftTokens, tokens } from "@/components/transfer/token-helper";
 import TransferNFT from "@/components/transfer/TransferNFT.vue";
 import ControllerModule from "@/modules/controllers";
 import { ALLOWED_VERIFIERS, ALLOWED_VERIFIERS_ERRORS, STATUS, STATUS_TYPE, TransferType } from "@/utils/enums";
-import { debounceAsyncValidator, delay, ruleVerifierId } from "@/utils/helpers";
+import { delay, ruleVerifierId } from "@/utils/helpers";
 import { SolAndSplToken } from "@/utils/interfaces";
 
 const { t } = useI18n();
 
 const snsError = ref("Account Does Not Exist");
 const isOpen = ref(false);
 const transferType = ref<TransferType>(ALLOWED_VERIFIERS[0]);
-const transferTo = ref<string>("");
+let timeoutId: ReturnType<typeof setTimeout>;
+const transferToInternal = ref("");
```

Comment:
is testing ->transfer-> fill up all values
1. add the address: 'butterchicken.sol'
2. click the transfer button
Expected: Owner is an escrow account, Actual-> goes through
Now,
Repeat the same process, but before filling the address, change the transfer type to "domain", then fill the address butterchicken.sol'.
This gives error as expected


This was because of the race condition, that which of the two - the validator or watcher, will run first.
By using the getter and setters, I am first changing the type of transfer as per domain, then setting the value to ref, and then the validator runs, giving the result as expected.